src/lib/subsys/subsys.h

```diff
@@ -70,6 +70,12 @@ typedef struct subsys_fns_t {
    */
   void (*postfork)(void);
 
+  /**
+   * Initialize any thread-local resources held by this subsystem. Called
+   * after the subsystem's global components are initialized.
```

Comment:
Is the idea that this should be invoked on every new thread that we spawn? If so we should document it.